docs/pages/docs/walkthroughs/getting-started-with-create-keystone-app.mdx

```diff
@@ -56,72 +56,56 @@ npx create-keystone-app
 
 ## Naming your app
 
-The CLI will ask you to name your new Keystone app and will create a new SQLite database for you.
+The CLI will ask you to name your app. Once named, it will create a new SQLite database.
 
 ?> You can switch to another database such as PostgreSQL once your project is created, check out the docs on [Database Setup](https://keystonejs.com/docs/apis/config#db).
 
 ## Opening your shiny new Admin UI
 
-![The AdminUI of Keystone showing the two lists: User and Posts](/assets/walkthroughs/getting-started/adminui.png)
-
-After running `create-keystone-app` you can now `cd` into the folder that was created for you and start Keystone.
+You can now `cd` into the folder that was created for you and start Keystone:
 
 ```sh
 cd my-app
 yarn dev
 ```
 
-This will generate the Admin UI pages via [Next.js](https://nextjs.org/) on [http://localhost:3000](http://localhost:3000).
+This will generate the Admin UI pages via [Next.js](https://nextjs.org/) on [http://localhost:3000](http://localhost:3000). When you visit the Admin UI for the first time you will be presented with a handy screen that asks you to create a user:
 
 ![The welcome screen giving you the ability the create a new user to log into the AdminUI](/assets/walkthroughs/getting-started/welcome-screen.png)
 
-When you visit the Admin UI for the first time you will be presented with a handy screen that asks you to create a user.
-Go ahead and create your first user.
-The email address and password will be used as the login details when you next need to login to the Admin UI.
+Go ahead and create your first user. The email address and password will be used to login to Keystone’s Admin UI. Once you've created your user, you’ll be logged in to a new Keystone Admin UI that comes with two [lists](/docs/apis/config#lists).
+From here you can explore and interact with the data in your system, and understand how Keystone’s schema relates to your GraphQL API which you can explore at [http://localhost:3000/api/graphql](http://localhost:3000/api/graphql).
 
-Once you've created the user you can see your new Keystone Admin UI that comes with two visible lists.
-You can use the Admin UI to explore and interact with the data in your system.
+![The AdminUI of Keystone showing the two lists: User and Posts](/assets/walkthroughs/getting-started/adminui.png)
 
 ## Output
 
-We now have some files in the newly generated folder. The two most important once you want to get into are `keystone.ts` and `schema.ts`.
+Keytone creates the following files in your newly generated folder. The most important ones are [`keystone.ts`](/docs/apis/config) and [`schema.ts`](/docs/apis/Schema).
 
 ```sh
 .
 ├── auth.ts        # Authentication configuration for Keystone
-├── keystone.ts    # The main entry file for Keystone
+├── keystone.ts    # Where you configure your project
```

Comment:
I think the words 'entry file' are really helpful, but weak opinion

Reply:
Fixed here:
https://github.com/keystonejs/keystone/pull/6586/commits/eba0328865531d11becd1d2bb58100ab1b0e236b